app/models/docker_registry.rb

```diff
@@ -3,6 +3,10 @@ class DockerRegistry < ActiveRecord::Base
   include Taxonomix
   include Encryptable
 
+  self.primary_key = 'id'
```

Comment:
I have no clue why I put it there... this should be the default, I'll try removing this